packages/patterns-react/src/patterns/LeadSpace/LeadSpace.js

```diff
@@ -97,7 +97,9 @@ const LeadSpace = ({
     <section
       data-autoid={`${stablePrefix}--leadspace`}
       className={className(variation, theme)}>
-      <div className={`${prefix}--leadspace__container`}>
+      <div
+        data-autoid={`${stablePrefix}--leadspace__container`}
```

Comment:
This here seems kinda excessive to me, we already have an autoid in for the `section` element above

Reply:
Yeah, I see your point. We already have it at parent level. And I just talked with Ke and confirmed that they need `data-autoid` on all optional fields and it's immediate parent.